packages/payload/src/collections/operations/create.ts

```diff
@@ -43,6 +44,7 @@ export type Arguments<TSlug extends CollectionSlug> = {
   disableTransaction?: boolean
   disableVerificationEmail?: boolean
   draft?: boolean
+  duplicateFromId?: number | string
```

Comment:
incorrect casing on "id", should be `duplicateFromID`

Reply:
updated!